README.rst

```diff
@@ -178,6 +153,46 @@ is ``helloworld`` in our case. Indeed:
    Hello John Doe!
    Hello Jane Doe!
 
+Local testing with CWL
+=========================
+
+Let us test whether the CWL workflow execution works locally as well.
+
+To prepare the execution, we are creating a working directory called ``cwl-local-run`` which will contain both
+``inputs`` and ``code`` directory content. Also, we need to copy the workflow input file:
+
+.. code-block:: console
+
+   $ mkdir cwl-local-run
+   $ cd cwl-local-run
+   $ cp ../code/* ../inputs/* ../workflow/cwl/helloworld-job.yml .
+
+We can now run the corresponding commands locally as follows:
+
+.. code-block:: console
+
+   $ cwltool --quiet --outdir="../outputs" ../workflow/cwl/helloworld.cwl helloworld-job.yml
+
+    {
+        "result": {
+            "checksum": "sha1$280335176499b850a4c0f46f16f31ee4cbd36754",
+            "basename": "greetings.txt",
+            "location": "file:///path/to/reana-demo-helloworld/outputs/greetings.txt",
+            "path": "/path/to/reana-demo-helloworld/outputs/greetings.txt",
+            "class": "File",
+            "size": 32
+        }
+    }
+
+Checking the output
+
+.. code-block:: console
+
+   $ cat outputs/greetings.txt
```

Comment:
That would be `cat ../outputs/greetings.txt` since you use outer directory... Either update the path, or perhaps we can do the output simply inside `cwl-local-run`?   

(But I can also update the path after merging since we shall have some verbosity updates...)